server/const.go

```diff
@@ -121,4 +121,10 @@ const (
 
 	// DEFAULT_MAX_CLOSED_CLIENTS
 	DEFAULT_MAX_CLOSED_CLIENTS = 10000
+
+	// DEFAULT_MAX_ACCOUNT_AE_RESPONSE_MAPS
+	DEFAULT_MAX_ACCOUNT_AE_RESPONSE_MAPS = 100000
+
+	// DEFAULT_TTL_AE_RESPONSE_MAP
+	DEFAULT_TTL_AE_RESPONSE_MAP = 60 * time.Second * 10 // 10 minutes
```

Comment:
can use `10 * time.Minute` to define duration too

Reply:
Ah yes thx.